public/templates/agents/agents-fim.html

```diff
@@ -66,7 +66,30 @@
                 </md-card-content>
             </md-card>
        </div>
-
-
-
+       
+    <div layout="row" class="wz-padding-top-0">
```

Comment:
What happens if our agent has no alerts for FIM? the table should be still visible for the user.